src/v2/api/controllers/AuthController.ts

```diff
@@ -143,6 +143,18 @@ export class AuthController {
     return this.authService.refresh(req.user);
   }
 
+  @ApiBearerAuth()
+  @ApiOkResponse({
+    type: AuthLoginResponse,
+  })
+  @ApiBadRequestResponse({
```

Comment:
You've missed 2 InvalidBodyExceptions and UnauthorizedException. Go to UpdatePasswordDTO and validateUser method to find these bad responses. Also InvalidEntityIdException must be "user with such id is not found"

Reply:
Done